definitions/npm/axios_v0.18.x/flow_v0.94.x-/axios_v0.18.x.js

```diff
@@ -0,0 +1,162 @@
+declare module 'axios' {
```

Comment:
Please lower the scope of one version,  the breaking change was introduced in Flow 0.93.  
`axios_v0.18.x/flow_v0.93.x-/axios_v0.18.x.js`

Reply:
Ah humm sorry, and also re-scope the previous one. 

Rename without change
 `axios_v0.18.x/flow_v0.80.x-/axios_v0.18.x.js` --->  `axios_v0.18.x/flow_v0.80.x-v0.92.x/axios_v0.18.x.js`

New file with fixed definitions
 `axios_v0.18.x/flow_v0.93.x-/axios_v0.18.x.js`